tests/expected/function-contract/history/block.expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
These should generally also list the important (i.e. specifically relevant to the particular test case) as expected. The reason is that then we also ensure that the test case is not vacuous.